.github/workflows/pr.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           cache: maven
 
       - name: build with maven
-        run: mvn clean install
+        run: mvn clean install -Dgithub.repo=${{ github.repository }} -Dgit.sha.commit=${{ github.event.pull_request.head.sha }}
```

Comment:
I don't think we need to pass these properties here any longer as we're now using defaults.
Also, this ensures the `mvn clean install` works as it is.

Reply:
That fails without passing such properties as quete reports `io.quarkus.qute.TemplateException: Rendering error in template [base.html] line 19: Property "github.repo" not found on the base object "null" in expression {config:github.repo}`

Reply:
According to Martin Kouba `{config:github.repo}` does not read the default value from ApplicationProperties but it attempts to read the config value directly :-(

Reply:
Should be fixed now: ccee6b0